index.js

```diff
@@ -319,6 +319,10 @@ function createFocusTrap(element, userOptions) {
       node.select();
     }
   }
+
+  function delay(fn) {
```

Comment:
`delay()` is also called at line 113 as part of the deactivation process. This change would affect that also.

Plus, here, we're defaulting to `false`, which means no delay, while we need to default to `true`, which means to delay, thus preserving the existing behavior.

Reply:
I would recommend we leave `delay()` where it's currently defined, without modifying it, and just an an IF statement around line 140 where the ELSE just calls `tryFocus(getInitialFocusNode());` immediately.

Reply:
@stefcameron Done.